examples/common/chip-app-server/QRCodeUtil.cpp

```diff
@@ -35,16 +35,16 @@ void PrintQRCode(chip::RendezvousInformationFlags rendezvousFlags)
 
     if (GetQRCode(setupPinCode, QRCode, rendezvousFlags) == CHIP_NO_ERROR)
     {
-        ChipLogProgress(AppServer, "SetupPINCode: [%" PRIu32 "]", setupPinCode);
-        ChipLogProgress(AppServer, "SetupQRCode:  [%s]", QRCode.c_str());
-
         chip::Platform::ScopedMemoryBuffer<char> qrCodeBuffer;
         const size_t qrCodeBufferMaxSize = 3 * QRCode.size() + 1;
         qrCodeBuffer.Alloc(qrCodeBufferMaxSize);
+
+        ChipLogProgress(AppServer, "SetupPINCode: [%" PRIu32 "]", setupPinCode);
+        ChipLogProgress(AppServer, "SetupQRCode:  [%s]", QRCode.c_str());
```

Comment:
Why move these two log lines?

Reply:
Just wanted to regroup those code lines thogether closer to the other Chip log process because they are all in linked to printing QRCode information